test/e2e-go/restAPI/restClient_test.go

```diff
@@ -1907,7 +1884,7 @@ int 1`
 	ops, err := logic.AssembleString(prog)
 	a.NoError(err)
 	approval := ops.Program
-	ops, err = logic.AssembleString("#pragma version 8; int 1")
```

Comment:
I'm seeing that `;` does not act as a newline if it's immediately after a `#pragma version` line, which is why I made this change

Reply:
Right, for better or worse, we treat "directives" differently than opcodes, and they remain line oriented. (Probably this is a silly holdover from me thinking about them like `cpp` directives.)

Reply:
should be resolved by #5400